clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -327,21 +329,22 @@ static KafkaAdminClient createInternal(AdminClientConfig config, TimeoutProcesso
     static KafkaAdminClient createInternal(AdminClientConfig config, KafkaClient client, Metadata metadata, Time time) {
         Metrics metrics = null;
         String clientId = generateClientId(config);
+        LogContext logContext = new LogContext("[KafkaAdminClient clientId=" + clientId + "] ");
```

Comment:
Maybe we should extract a static method `createLogContext` to avoid duplicating the logic. Also, `AdminClient` (instead of `KafkaAdminClient`) is shorter and provides enough context.

Reply:
@ijuma 0847add